httomo/runner/loader.py

```diff
@@ -166,12 +167,16 @@ def __init__(
         slicing_dim: Literal[0, 1, 2],
         comm: MPI.Comm,
     ) -> None:
+        if slicing_dim != 0:
+            raise NotImplementedError("Only slicing dim 0 is currently supported")
+
```

Comment:
you should add a quick test for this - unless you fixed it in the previews branch to support all slicing dims?

Reply:
Done with [486c9e9](https://github.com/DiamondLightSource/httomo/pull/202/commits/486c9e99e795b629f92d360c0e56e13548093d6d)